modules/silicon_design/main.tf

```diff
@@ -170,7 +166,7 @@ resource "google_project_iam_member" "sa_p_notebook_permissions" {
 resource "google_project_iam_member" "sa_p_cloudbuild_permissions" {
   for_each = toset(local.cloudbuild_sa_project_roles)
   project  = local.project.project_id
-  member   = "serviceAccount:${data.google_project.project.number}@cloudbuild.gserviceaccount.com"
+  member   =  var.create_project ? "serviceAccount:${local.project.project_number}@cloudbuild.gserviceaccount.com" : "serviceAccount:${local.project.number}@cloudbuild.gserviceaccount.com"
```

Comment:
what about doing this in a `local.project_number` local, so that we can refer to the project number uniformly in the rest of the recipe?

Reply:
fiexed with https://github.com/GoogleCloudPlatform/rad-lab/pull/50/commits/51459bbf0c01b9074d20fe655492548bd64052b8